sonar-python-plugin/src/test/java/org/sonar/plugins/python/IpynbNotebookParserTest.java

```diff
@@ -138,6 +138,10 @@ void testParseNotebookSingleLine() throws IOException {
     assertThat(result.locationMap()).hasSize(5);
     assertThat(result.contents()).hasLineCount(5);
     // position of variable t
-    assertThat(result.locationMap().get(4).column()).isEqualTo(451);
+    assertThat(result.locationMap().get(4).column()).isEqualTo(452);
```

Comment:
Here the column count should start at 0
as stated in sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java line 225. The t should be the 451st character no? Why did you need to change the count? Was the issue only related to the first line? in which case?

Reply:
I checked which character it was manually, and at 451 the location is between the `\` and the `n`, which was incorrect.